### PR TITLE
DoS Status

### PR DESCRIPTION
**Pull request by pasmant**
Add for DoS the dashboard page and api, in the case '-nginx-status' cli argument is enable.

- [V] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [V] I have added tests that prove my fix is effective or that my feature works
- [V] I have checked that all unit tests pass after adding my changes
- [V] I have updated necessary documentation
- [V] I have rebased my branch onto main
- [V] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/logging-and-monitoring/status-page.md [hunk 1]
```diff
@@ -9,7 +9,10 @@ docs: "DOCS-615"
 ---
 
 
-NGINX comes with a status page that reports basic metrics about NGINX called the [stub status](https://nginx.org/en/docs/http/ngx_http_stub_status_module.html). NGINX Plus comes with a [dashboard](https://docs.nginx.com/nginx/admin-guide/monitoring/live-activity-monitoring/) that reports key load-balancing and performance metrics. This doc shows how to get access to the stub status/dashboard.
+NGINX comes with a status page that reports basic metrics about NGINX called the [stub status](https://nginx.org/en/docs/http/ngx_http_stub_status_module.html).
+NGINX Plus comes with a [dashboard](https://docs.nginx.com/nginx/admin-guide/monitoring/live-activity-monitoring/) that reports key load-balancing and performance metrics.
+NGINX App Protect Dos comes with a [dashboard](https://docs.nginx.com/nginx-app-protect-dos/monitoring/live-activity-monitoring/)  that shows the status and information of the protected objects.
```
Comment (lucacome) [inline docs/content/logging-and-monitoring/status-page.md:14]:
```suggestion
NGINX App Protect DoS comes with a [dashboard](https://docs.nginx.com/nginx-app-protect-dos/monitoring/live-activity-monitoring/) that shows the status and information of the protected objects.
```

File: docs/content/logging-and-monitoring/status-page.md [hunk 2]
```diff
@@ -40,9 +43,10 @@ To access the dashboard:
     $ kubectl port-forward <nginx-plus-ingress-pod> 8080:8080 --namespace=nginx-ingress
     ```
 1. Open your browser at http://127.0.0.1:8080/dashboard.html to access the dashboard.
+1. App Protect Dos: Open your browser at http://127.0.0.1:8080/dashboard-dos.html to access the dashboard.
```
Comment (lucacome) [inline docs/content/logging-and-monitoring/status-page.md:46]:
```suggestion
1. App Protect DoS: Open your browser at http://127.0.0.1:8080/dashboard-dos.html to access the dashboard.
```

File: docs/content/logging-and-monitoring/status-page.md [hunk 3]
```diff
@@ -40,9 +43,10 @@ To access the dashboard:
...
 
 If you want to access the dashboard externally (without `kubectl port-forward`):
 1. Configure `-nginx-status-allow-cidrs` command-line argument with IP/CIDR blocks for which you want to allow access to the dashboard. By default, the access is allowed for `127.0.0.1,::1`.
 1. Use the IP/port through which the Ingress Controller pod/pods are available to connect the dashboard at the `/dashboard.html` path.
 
-**Note**: The [API](https://nginx.org/en/docs/http/ngx_http_api_module.html), which the dashboard uses to get the metrics, is also accessible: use the `/api` path. Note that the API is configured in the read-only mode.
+**Note**: The [API](https://nginx.org/en/docs/http/ngx_http_api_module.html), which the dashboard uses to get the metrics, is also accessible: use the `/api` path. For App Protect Dos use the `/api/dos` path. Note that the API is configured in the read-only mode.
```
Comment (lucacome) [inline docs/content/logging-and-monitoring/status-page.md:52]:
```suggestion
**Note**: The [API](https://nginx.org/en/docs/http/ngx_http_api_module.html), which the dashboard uses to get the metrics, is also accessible: use the `/api` path. For App Protect DoS use the `/api/dos` path. Note that the API is configured in the read-only mode.
```

**Review by lucacome — CHANGES_REQUESTED** [review]

**Comment by jjngx** [comment]
@pasmant looks ok, could you please fill info required in the PR template, and description of performed tests?

**Pull request merged**